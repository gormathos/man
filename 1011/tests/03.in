sol_id;user_id;task_id;score
182587;1006;2001;0.00
182587;1030;2001;27.66
182592;1024;2003;7.69
182599;1027;2001;8.51
182599;1021;2001;2.13
182605;1028;2004;46.43
182608;1015;2004;32.14
182612;1006;2003;17.95
182616;1006;2001;38.30
182623;1011;2003;84.62
182625;1012;2004;42.86
182633;1013;2003;35.90
182637;1004;2001;74.47
182638;1008;2003;58.97
182648;1020;2001;17.02
182651;1032;2001;48.94
182652;1031;2002;82.86
182655;1012;2002;14.29
182655;1010;2001;6.38
182656;1003;2002;37.14
182658;1028;2002;74.29
182666;1001;2004;21.43
182674;1011;2002;0.00
182674;1010;2001;34.04
182675;1022;2004;7.14
182684;1013;2002;2.86
182690;1029;2001;87.23
182691;1027;2001;14.89
182701;1020;2003;15.38
182711;1009;2001;2.13
182720;1032;2004;60.71
182722;1010;2002;14.29
182730;1015;2001;40.43
182731;1034;2001;4.26
182735;1016;2001;29.79
182736;1017;2002;68.57
182736;1004;2004;21.43
182744;1007;2004;50.00
182748;1016;2001;10.64
182757;1019;2001;68.09
182760;1010;2003;71.79
182765;1023;2004;28.57
182766;1033;2004;53.57
182767;1017;2001;19.15
182768;1031;2001;29.79
182774;1007;2001;29.79
182780;1015;2004;10.71
182789;1006;2002;0.00
182792;1006;2004;0.00
182793;1016;2003;17.95
182803;1030;2004;50.00
182806;1023;2002;45.71
182816;1029;2003;7.69
182821;1033;2002;100.00
182826;1008;2002;11.43
182833;1034;2002;71.43
182836;1011;2002;28.57
182846;1016;2004;14.29
182847;1019;2003;28.21
182849;1019;2004;39.29
182850;1021;2003;23.08
182857;1010;2002;17.14
182867;1008;2004;50.00
182875;1027;2003;87.18
182885;1025;2004;50.00
182889;1034;2002;57.14
182893;1031;2001;40.43
182896;1025;2001;29.79
182905;1014;2004;89.29
182907;1021;2002;20.00
182909;1029;2001;8.51
182917;1026;2001;10.64
182919;1003;2003;61.54
182927;1008;2001;72.34
182936;1032;2002;37.14
182941;1004;2001;55.32
182944;1022;2001;2.13
182949;1005;2002;0.00
182957;1010;2003;35.90
182958;1003;2001;78.72
182958;1028;2003;46.15
182965;1009;2002;85.71
182967;1017;2001;87.23
182971;1030;2003;25.64
182981;1014;2001;42.55
182990;1017;2002;5.71
182995;1029;2004;28.57
182997;1020;2004;17.86
183000;1003;2001;100.00
183005;1015;2002;0.00
183015;1009;2003;87.18
183024;1024;2003;15.38
183031;1011;2003;5.13
183037;1025;2002;11.43
183043;1021;2003;0.00
183053;1027;2001;40.43
183057;1029;2002;34.29
183059;1024;2004;32.14
183063;1027;2004;96.43
183070;1034;2001;38.30
183074;1022;2003;5.13
183078;1015;2002;71.43
183079;1017;2003;15.38
183081;1030;2001;27.66
183087;1019;2003;76.92
183093;1026;2002;5.71
183101;1027;2003;15.38
183103;1030;2002;2.86
183110;1024;2002;65.71
183119;1004;2001;42.55
183122;1015;2001;10.64
183126;1016;2002;28.57
183136;1005;2003;15.38
183136;1013;2003;7.69
183140;1014;2001;17.02
183149;1001;2003;41.03
183150;1027;2002;40.00
183160;1022;2001;29.79
183162;1015;2002;57.14
183165;1024;2002;8.57
183167;1026;2003;41.03
183172;1008;2003;12.82
183175;1031;2001;53.19
183179;1005;2004;60.71
183186;1028;2004;17.86
183192;1023;2004;3.57
183200;1024;2003;53.85
183203;1024;2004;60.71
183207;1034;2001;59.57
183213;1017;2004;14.29
183222;1011;2001;6.38
183231;1008;2004;7.14
183232;1019;2004;17.86
183233;1022;2001;27.66
183236;1034;2003;20.51
183241;1005;2001;34.04
183243;1021;2001;12.77
183246;1011;2003;33.33
183247;1001;2001;46.81
183252;1006;2004;67.86
183256;1026;2003;2.56
183257;1010;2003;2.56
183261;1009;2001;40.43
183268;1031;2003;53.85
183270;1029;2004;7.14
183270;1001;2003;0.00
183273;1019;2002;11.43
183278;1018;2001;23.40
183280;1031;2003;61.54
183286;1031;2003;0.00
183295;1013;2001;36.17
183297;1004;2003;5.13
183303;1018;2004;17.86
183307;1014;2003;17.95
183308;1030;2004;75.00
183313;1005;2003;0.00
183313;1029;2002;5.71
183323;1025;2003;15.38
183333;1034;2004;3.57
183336;1015;2001;0.00
183343;1022;2004;89.29
183350;1008;2001;23.40
183359;1026;2004;21.43
183368;1027;2002;54.29
183372;1003;2004;53.57
183381;1009;2003;7.69
183387;1019;2003;35.90
183388;1028;2001;72.34
183397;1017;2003;33.33
183401;1002;2004;53.57
183403;1034;2004;0.00
183410;1016;2002;62.86
183416;1027;2002;11.43
183419;1008;2002;8.57
183419;1017;2001;2.13
183423;1018;2001;34.04
183425;1020;2002;54.29
183426;1013;2001;46.81
183427;1008;2002;22.86
183431;1027;2003;2.56
183436;1009;2001;17.02
183444;1014;2001;65.96
183452;1033;2003;79.49
183453;1034;2004;71.43
183456;1025;2004;0.00
183463;1026;2003;30.77
183471;1016;2003;2.56
183478;1001;2004;39.29
183488;1023;2001;19.15
183496;1018;2001;97.87
183506;1013;2004;10.71
183515;1012;2001;10.64
183518;1008;2003;94.87
183518;1005;2001;46.81
183522;1020;2001;80.85
183528;1025;2001;17.02
183531;1023;2001;17.02
183535;1005;2004;25.00
183545;1032;2002;5.71
183554;1001;2002;5.71
183563;1033;2001;25.53
183573;1014;2002;22.86
183578;1022;2002;8.57
183582;1015;2004;28.57
183588;1025;2002;17.14
183596;1024;2001;36.17
183600;1011;2002;82.86
183601;1005;2004;25.00
183601;1032;2002;68.57
183603;1026;2001;10.64
183607;1013;2002;0.00
183615;1033;2003;25.64
183617;1009;2004;35.71
183623;1020;2003;35.90
183626;1010;2002;45.71
183630;1023;2003;2.56
183639;1032;2003;30.77
183646;1001;2001;0.00
183655;1018;2004;21.43
183657;1022;2002;0.00
183657;1007;2001;87.23
183660;1016;2004;0.00
183665;1024;2001;59.57
183675;1018;2003;0.00
183680;1006;2003;7.69
183685;1005;2002;5.71
183687;1012;2002;77.14
183693;1021;2004;67.86
183701;1007;2002;88.57
183704;1032;2003;71.79
183705;1030;2004;85.71
183712;1032;2004;3.57
183721;1020;2002;2.86
183729;1018;2003;66.67
183732;1012;2003;23.08
183736;1001;2002;0.00
183738;1023;2003;10.26
183739;1030;2001;53.19
183746;1031;2004;67.86
183751;1006;2002;97.14
183757;1019;2002;11.43
183765;1022;2003;92.31
183766;1029;2003;10.26
183775;1009;2004;39.29
183779;1003;2001;55.32
183782;1017;2004;7.14
183791;1023;2004;21.43
183794;1024;2004;75.00
183804;1011;2001;6.38
183804;1004;2003;89.74
183811;1007;2003;5.13
183820;1015;2003;43.59
183827;1009;2002;45.71
183834;1010;2004;67.86
183842;1024;2002;57.14
183846;1017;2002;17.14
183856;1032;2004;39.29
183858;1002;2001;17.02
183858;1018;2002;42.86
183868;1021;2004;57.14
183870;1002;2004;75.00
183874;1009;2002;0.00
183884;1033;2004;96.43
183892;1021;2002;0.00
183892;1016;2002;77.14
183893;1009;2004;60.71
183896;1023;2002;2.86
183899;1015;2003;17.95
183908;1014;2003;43.59
183910;1012;2002;25.71
183919;1001;2001;36.17
183920;1026;2004;0.00
183927;1017;2004;60.71
183929;1003;2004;0.00
183929;1030;2002;34.29
183934;1013;2004;46.43
183939;1006;2003;0.00
183941;1003;2003;76.92
183944;1025;2003;58.97
183954;1011;2004;57.14
183964;1004;2004;21.43
183964;1032;2001;57.45
183970;1019;2001;59.57
183976;1020;2001;34.04
183986;1025;2001;63.83
183989;1003;2002;0.00
183999;1032;2001;72.34
183999;1002;2004;14.29
184004;1028;2003;53.85
184006;1013;2001;76.60
184007;1029;2001;95.74
184008;1012;2003;94.87
184018;1021;2004;71.43
184025;1025;2003;71.79
184029;1001;2004;53.57
184033;1005;2001;8.51
184037;1004;2002;25.71
184047;1003;2002;28.57
184055;1022;2002;77.14
184065;1021;2003;58.97
184067;1031;2004;10.71
184074;1020;2004;0.00
184075;1028;2001;95.74
184079;1005;2003;30.77
184088;1023;2003;33.33
184093;1012;2004;7.14
184093;1020;2003;12.82
184103;1027;2004;0.00
184104;1004;2002;40.00
184109;1033;2001;2.13
184111;1028;2003;2.56
184114;1007;2002;68.57
184120;1018;2004;3.57
184130;1031;2002;48.57
184140;1026;2004;96.43
184143;1012;2001;4.26
184143;1004;2002;62.86
184152;1017;2003;48.72
184154;1033;2003;76.92
184163;1018;2003;7.69
184164;1009;2003;15.38
184169;1034;2003;10.26
184172;1007;2004;3.57
184172;1004;2004;0.00
184181;1007;2001;17.02
184187;1013;2002;37.14
184192;1016;2003;30.77
184192;1013;2004;100.00
184199;1030;2002;0.00
184206;1028;2002;42.86
184209;1028;2002;82.86
184216;1029;2003;2.56
184218;1012;2001;6.38
184224;1006;2002;68.57
184227;1005;2002;34.29
184230;1011;2004;53.57
184237;1002;2003;43.59
184242;1014;2002;60.00
184251;1022;2003;2.56
184251;1002;2001;2.13
184257;1029;2002;17.14
184259;1018;2002;60.00
184260;1010;2001;59.57
184261;1012;2004;67.86
184261;1014;2004;35.71
184263;1023;2001;31.91
184269;1019;2001;12.77
184276;1007;2002;0.00
184282;1023;2002;91.43
184286;1010;2004;71.43
184292;1001;2003;5.13
184295;1031;2002;2.86
184301;1033;2004;35.71
184308;1024;2001;40.43
184318;1002;2002;0.00
184321;1014;2003;61.54
184323;1007;2003;5.13
184325;1028;2004;64.29
184335;1003;2003;89.74
184340;1008;2004;7.14
184345;1002;2001;14.89
184346;1026;2002;51.43
184356;1025;2002;91.43
184360;1032;2003;48.72
184367;1013;2003;56.41
184369;1001;2002;17.14
184370;1034;2002;65.71
184380;1014;2004;7.14
184380;1015;2003;7.69
184388;1012;2003;30.77
184392;1016;2004;35.71
184392;1016;2001;8.51
184396;1025;2004;0.00
184398;1031;2004;21.43
184403;1034;2003;23.08
184413;1011;2001;4.26
184423;1002;2003;64.10
184426;1028;2001;8.51
184429;1020;2004;17.86
184433;1026;2001;21.28
184443;1002;2003;0.00
184445;1004;2003;51.28
184451;1021;2001;4.26
184461;1018;2002;2.86
184471;1011;2004;50.00
184474;1030;2003;30.77
184474;1022;2004;0.00
184479;1010;2004;67.86
184483;1029;2004;39.29
184486;1019;2004;28.57
184495;1027;2004;14.29
184495;1008;2001;12.77
184495;1026;2002;80.00
184503;1019;2002;2.86
184508;1033;2001;0.00
184512;1006;2001;51.06
184522;1033;2002;8.57
184530;1020;2002;8.57
184537;1030;2003;2.56
184540;1033;2002;31.43
184542;1021;2002;11.43
184548;1006;2004;7.14
184549;1007;2003;43.59
184550;1003;2004;3.57
184558;1007;2004;85.71
184566;1014;2002;82.86
184575;1002;2002;20.00
184583;1002;2002;40.00
