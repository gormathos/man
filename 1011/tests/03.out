1001: Participant [39.64%]
1002: Participant [49.03%]
1003: Very Good [70.11%]
1004: Very Good [62.12%]
1005: Participant [43.14%]
1006: Participant [58.50%]
1007: Excellent [76.28%]
1008: Very Good [60.02%]
1009: Very Good [68.51%]
1010: Very Good [62.12%]
1011: Participant [57.75%]
1012: Very Good [62.63%]
1013: Very Good [67.54%]
1014: Excellent [74.91%]
1015: Participant [46.90%]
1016: Participant [43.35%]
1017: Very Good [66.31%]
1018: Very Good [61.49%]
1019: Participant [48.93%]
1020: Participant [47.22%]
1021: Participant [40.79%]
1022: Very Good [72.13%]
1023: Participant [46.31%]
1024: Very Good [63.53%]
1025: Very Good [69.26%]
1026: Participant [59.69%]
1027: Very Good [69.58%]
1028: Excellent [74.19%]
1029: Participant [44.89%]
1030: Participant [50.99%]
1031: Very Good [66.36%]
1032: Very Good [68.35%]
1033: Excellent [75.36%]
1034: Participant [56.38%]
